app/scripts/metamask-controller.js

```diff
@@ -4110,7 +4110,7 @@ export default class MetamaskController extends EventEmitter {
     const { tokens } = this.tokensController.state;
 
     const staticTokenListDetails =
-      STATIC_MAINNET_TOKEN_LIST[address.toLowerCase()] || {};
+      STATIC_MAINNET_TOKEN_LIST[address?.toLowerCase()] || {};
```

Comment:
Do we know how to reproduce this one?  Don't think we'd expect an undefined address to be passed here.  But I also didn't see a useful call stack to determine who passed it here

Reply:
I could not repro either, the fct is called in multiple places, i suspected this [line](https://github.com/MetaMask/metamask-extension/blob/e76a149e14c119eba79383c791ed17da5ec18d98/shared/modules/transaction.utils.ts#L258) but could not repro